core/vm/lfvm/gas.go

```diff
@@ -6,24 +6,66 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/ethereum/go-ethereum/params"
 	"github.com/holiman/uint256"
 )
 
 const UNKNOWN_GAS_PRICE = 999999
 
 var static_gas_prices = [NUM_OPCODES]uint64{}
+var static_gas_prices_berlin = [NUM_OPCODES]uint64{}
 
 func init() {
+	var gp uint64
 	for i := 0; i < int(NUM_OPCODES); i++ {
-		static_gas_prices[i] = getStaticGasPriceInternal(OpCode(i))
+		gp = getStaticGasPriceInternal(OpCode(i))
+		static_gas_prices[i] = gp
+		static_gas_prices_berlin[i] = gp
 	}
+	initBerlinGasPrice()
 }
 
-func getStaticGasPrice(op OpCode) uint64 {
+func initBerlinGasPrice() {
+	// Changed static gas prices
+	static_gas_prices_berlin[SLOAD] = 0
+	static_gas_prices_berlin[EXTCODECOPY] = 100
+	static_gas_prices_berlin[EXTCODESIZE] = 100
+	static_gas_prices_berlin[EXTCODEHASH] = 100
+	static_gas_prices_berlin[BALANCE] = 100
+	static_gas_prices_berlin[CALL] = 100
+	static_gas_prices_berlin[CALLCODE] = 100
+	static_gas_prices_berlin[STATICCALL] = 100
+	static_gas_prices_berlin[DELEGATECALL] = 100
+	static_gas_prices_berlin[SELFDESTRUCT] = 5000
+
+	// Changed dynamic gas prices for these opcodes
+	// implemented directly in instructions
+	//
+	// jt[SSTORE].dynamicGas = gasSStoreEIP2929
+	// jt[SLOAD].dynamicGas = gasSLoadEIP2929
+	// jt[EXTCODECOPY].dynamicGas = gasExtCodeCopyEIP2929
+	// jt[EXTCODESIZE].dynamicGas = gasEip2929AccountCheck
+	// jt[EXTCODEHASH].dynamicGas = gasEip2929AccountCheck
+	// jt[BALANCE].dynamicGas = gasEip2929AccountCheck
+	// jt[CALL].dynamicGas = gasCallEIP2929
+	// jt[STATICCALL].dynamicGas = gasStaticCallEIP2929
+	// jt[DELEGATECALL].dynamicGas = gasDelegateCallEIP2929
+	// jt[SELFDESTRUCT].dynamicGas = gasSelfdestructEIP2929
+
+	// this instruction is not used sofar for LFVM
+	// jt[CALLCODE].constantGas = params.WarmStorageReadCostEIP2929
+	// jt[CALLCODE].dynamicGas = gasCallCodeEIP2929
```

Comment:
Do we still need the old table as a comment?

Reply:
removed